pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSource.java

```diff
@@ -19,6 +19,7 @@
 
 package org.apache.pulsar.io.kafka;
 
+import lombok.Getter;
```

Comment:
kafka source is natively pull source. So we should actually implement it as source<T>

Reply:
sure can we do that in a followup PR?